pkgs/development/tools/misc/debugedit/default.nix

```diff
@@ -0,0 +1,29 @@
+#TODO@deliciouslytyped The tool seems to unnecessarily force mutable access for the debugedit `-l` feature
+{fetchgit, lib, stdenv, autoreconfHook, pkgconfig, elfutils, help2man, util-linux}:
+stdenv.mkDerivation {
+  name = "debugedit";
+  version = "unstable-2021-07-05";
+
+  buildInputs = [ autoreconfHook pkgconfig elfutils help2man ];
```

Comment:
Probably only elfutil is a buildInput, the other ones are nativeBuildInputs. If unsure, add strictDeps = true.